R/transformations.R

```diff
@@ -0,0 +1,61 @@
+# common effect size transformations for using correlations input in selection models and PET-PEESE
+.maGetInputEs              <- function(dataset, options) {
+
+  if (!is.null(options[["effectDirection"]]) && options[["effectDirection"]] == "negative")
+    dataset[, options[["inputES"]]] <- dataset[, options[["inputES"]]] * - 1
+
+  return(switch(
+    options[["measures"]],
+    "general"     = dataset[, options[["inputES"]]],
+    "correlation" = .maTransformEs(dataset[, options[["inputES"]]], options[["muTransform"]])
+  ))
+}
+.maGetInputSe              <- function(dataset, options) {
+  return(switch(
+    options[["measures"]],
+    "general"     = dataset[, options[["inputSE"]]],
+    "correlation" = .maComputeSe(dataset[, options[["inputES"]]], dataset[, options[["inputN"]]], options[["muTransform"]])
+  ))
+}
+.maGetInputPVal            <- function(dataset, options) {
+  # weightfunc uses one-sided p-values as input!
+  if (options[["inputPVal"]] == "")
+    return(pnorm(.maGetInputEs(dataset, options) /.maGetInputSe(dataset, options), lower.tail = FALSE))
+  else
+    return(dataset[, .v(options[["inputPVal"]])])
```

Comment:
there is a `.v()` here

Reply:
ah, last forgotten one :D thanks, removed